datalad/support/annexrepo.py

```diff
@@ -13,30 +13,38 @@
 """
 
 from os import linesep
-from os.path import join as p_join, exists, normpath, isabs, commonprefix, relpath
+from os.path import join as opj, exists, normpath, isabs, commonprefix, relpath
 import logging
 
+from functools import wraps
+
 from ConfigParser import NoOptionError
 
-from gitrepo import GitRepo, files_decorator
+from gitrepo import GitRepo, normalize_paths
 from datalad.cmd import Runner as Runner
-from exceptions import CommandNotAvailableError, CommandError, FileNotInAnnexError, FileInGitError
+from exceptions import CommandNotAvailableError, CommandError, \
+    FileNotInAnnexError, FileInGitError
 
 lgr = logging.getLogger('datalad.annex')
 
 
-def _options_decorator(func):
+def kwargs_to_options(func):
```

Comment:
btw -- not sure yet if this is the way to go... if it would be the way -- decorator should adjust docstring of each decorated method to provide information that it is sucking out kwargs into options and doesn't allow options list being passed directly.